csharp/src/Google.Protobuf/Collections/MapField.cs

```diff
@@ -452,7 +452,17 @@ public void WriteTo(CodedOutputStream output, Codec codec)
             WriteContext.Initialize(output, out WriteContext ctx);
             try
             {
-                WriteTo(ref ctx, codec);
+                if (output.IsSerializationDeterministic())
```

Comment:
How do I unit-test this ? Is this for old proto2 format ?

Reply:
I'm not sure what you mean by that. Is *what* for proto2 format?